schemas/IDLs/org.gel.models.report.avro/3.1.0/CancerParticipant.avdl

```diff
@@ -91,12 +91,12 @@ record CancerSample {
     union {null, string} grade;
 
     /**
-    TMN version
+    TNM version
     */
     union {null, string} tnmStageVersion;
 
     /**
-    TMN stage grouping
+    TNM stage grouping
     */
     union {null, string} tmnStageGrouping;
```

Comment:
while we're at it shall we change `tmnStageGrouping` to `tnmStageGrouping` here ?